### ISSUE TITLE
vocoder training 질문드립니다.

### ISSUE THREAD

**Issue by yoonjeewoo** [hccho2/Tacotron-Wavenet-Vocoder#5]
안녕하세요 좋은 코드를 바탕으로 열심히 공부중인 학생입니다..!!

2019-10-04 13:38:15.119765: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1435] Adding visible gpu devices: 0
2019-10-04 13:38:15.457661: I tensorflow/core/common_runtime/gpu/gpu_device.cc:923] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-10-04 13:38:15.457699: I tensorflow/core/common_runtime/gpu/gpu_device.cc:929]      0
2019-10-04 13:38:15.457711: I tensorflow/core/common_runtime/gpu/gpu_device.cc:942] 0:   N
2019-10-04 13:38:15.457799: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1053] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10426 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:06:00.0, compute capability: 6.1)
Trying to restore saved checkpoints from ./logdir-wavenet/train/2019-10-04T13-37-34 ... No checkpoint found.
finally
Done

vocoder train시에
이런식으로 Done 이후에 계속 멈춰있는데 뭐가 문제일까요..?

**Comment by yoonjeewoo** [comment]
InvalidArgumentError (see above for traceback): Expected begin[1] in [0, 4800], but got 5147
         [[Node: wavenet/loss/Slice = Slice[Index=DT_INT32, T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"](wavenet/Reshape_1, wavenet/loss/Slice/begin, wavenet/loss/Slice/size)]]
         [[Node: optimizer/gradients/wavenet/dilated_stack/layer30/dilation_layer/conv_filter/conv1d/Conv2D_grad/ShapeN/_1789 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_15892...rad/ShapeN", tensor_type=DT_INT32, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

에러를 열어보니 이런 에러가 뜹니다.
